packages/Makefile

```diff
@@ -6,20 +8,18 @@ else
 	ONLY_PACKAGES=--only "$(PYODIDE_PACKAGES)"
 endif
 
+all: pyodide-build
+	PYODIDE_ROOT=$(PYODIDE_ROOT) python -m pyodide_build buildall . $(PYODIDE_ROOT)/build \
+	$(ONLY_PACKAGES) --n-jobs $${PYODIDE_JOBS:-4} \
+	--log-dir=./build-logs
 
-all: .artifacts/bin/pyodide-build
-	PYTHONPATH="$(PYODIDE_ROOT)/pyodide-build/"\
-		python -m pyodide_build buildall . ../build \
-		$(ONLY_PACKAGES) --n-jobs $${PYODIDE_JOBS:-4} \
-		--log-dir=./build-logs
-
-.artifacts/bin/pyodide-build: ../pyodide-build/pyodide_build/**
+pyodide-build: ../pyodide-build/pyodide_build/**
 	mkdir -p $(HOSTINSTALLDIR)
-	$(HOSTPYTHON) -m pip install -e ../pyodide-build --no-deps --prefix $(HOSTINSTALLDIR)
+	$(HOSTPYTHON) -m pip install -q --user --upgrade --force-reinstall ../pyodide-build
```

Comment:
We were using `pyodide-build` in a strange way during package build. We install it in `HOSTINSTALLDIR` but we don't use it. Instead, we just inject `pyodide-build` path to `PYTHONPATH`.

Reply:
I think it would still be preferable to use editable install.

Also the `--user` option should almost always be avoided in favor of using a virtualenv, so IMO we should't silently install stuff un userdir.

Reply:
Alright, I'll find a way to use editable install and use it directly. Note that there is an issue in `pyproject.toml + user mode + editable install` https://github.com/pyodide/pyodide/issues/2020#issuecomment-1049430878